src/permissions/models.rs

```diff
@@ -39,11 +39,12 @@ pub struct PermissionList {
 }
 
 pub enum PermissionRequest {
-    GetPermission(u64), //id of permission name searched
+    GetPermission(u64), //id of permission name searched permission
     CreatePermission(NewPermission), //new permission type of some name to be created
     UpdatePermission(u64, PartialPermission), //Contains id to be changed to new permission_name
     DeletePermission(u64),                    //if of permission to be deleted
     FirstPermission(String),
+    GetAllPossiblePermissions // get all possible permissions from sql table
```

Comment:
The request should be `GetPermissions` instead of `GetAllPossiblePermissions` to be more consistent